pallets/subtensor/src/lib.rs

```diff
@@ -949,6 +949,9 @@ pub mod pallet {
                     // Update total issuance value
                     TotalIssuance::<T>::put(TotalIssuance::<T>::get().saturating_add(*stake));
 
+                    // Update total stake value
+                    TotalStake::<T>::put(TotalStake::<T>::get().saturating_add(*stake));
+
```

Comment:
Might be good to leave a comment here mentioning that the genesis block was not produced using this. i.e. this was added afterwards, so the genesis state would be different for a new instance of the chain.